src/App.vue

```diff
@@ -468,8 +465,9 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
         this.fetchErrorDialog = true
         throw error // go to catch()
       }
-    } catch (e) {
+    } catch (error) {
       // errors should be handled above
+      console.error('Unhandled error in fetchData() =', error) // eslint-disable-line no-console
```

Comment:
This doesn't affect processing but displays the error (and stack trace) if something above didn't handle it.